### DISCUSSION TITLE
WebView2

### DISCUSSION THREAD

**Discussion by andersra** [MicrosoftEdge/WebView2Feedback#4805] [Q&A] [answered]
First of all what is required to get WebView2 installed on a Win 10 and Win 11? Can it be installed by a standard Windows Update?

If it doesn't come with Windows Updates can you then help to point to the package I need to include in my installer to ensure it's installed even on PC's that doesn't contain an Internet connection during installation.

Thanks
AHR

**Comment by victorhuangwq** [comment, marked as answer]
With respect to availability of WebView2 on Win 10 and Win11: https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution?tabs=dotnetcsharp#windows-11-devices-and-windows-10-devices

For offline situation, the standalone installer is what you are looking for: https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution?tabs=dotnetcsharp#understanding-the-options-at-the-runtime-download-page
